src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -16,37 +17,47 @@ void    showClInfo ();
 // main function
 int main(int argc, char* argv[])
 {
-    std::string             sInputFilePath,                 //!< file paths
-                            sOutputFilePath;
+    std::string sInputFilePath,                 //!< file paths
+                sOutputFilePath;
 
-    static const int        kBlockSize = 1024;
+    static const int kBlockSize = 1024;
 
-    clock_t                 time = 0;
+//    clock_t time = 0;
 
-    float                   **ppfAudioData = 0;
+    float **ppfAudioData = 0;
+    float **ppfProcessedData = 0;
 
-    CAudioFileIf            *phAudioFile = 0;
-    std::fstream            hOutputFile;
+    CAudioFileIf *phAudioFile = 0;
+    CAudioFileIf *hOutputFile = 0;
+    CCombFilterIf *pcCombFilter = 0;
     CAudioFileIf::FileSpec_t stFileSpec;
 
     showClInfo();
 
     //////////////////////////////////////////////////////////////////////////////
     // parse command line arguments
-    if (argc < 2)
+    if (argc < 4)
     {
         cout << "Missing audio input path!";
         return -1;
     }
     else
```

Comment:
It would be nicer if you can add test function here when command line arguments don't work very well.